math/postfix_calculator.cpp

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Please add comments in [doxygen](https://www.doxygen.nl/manual/docblocks.html) style. You can also refer to #807 and #840